docs/recorder/provider.swagger.yaml

```diff
@@ -2,7 +2,6 @@ swagger: "2.0"
 info:
   title: recorder/provider.proto
   version: version not set
-host: gateway.beta.wecasino.live
 consumes:
```

Comment:
revert this cause buf build doesn't get the example, dont figure out how to fix. so I mantaul add this one